.github/workflows/test_migrations.yaml

```diff
@@ -0,0 +1,246 @@
+          echo "CURRENT_HEAD=${current_head}" >> $GITHUB_ENV
+
+      - name: Checkout commit before the very first migration
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          ref: c48fa7f040aa83a494a365f80a8dffeaa0af9a1f
+          submodules: true
+
+      - name: Write SkyPortal configs
+        run: |
+          cat << EOF > config.yaml
+            database:
+              database: skyportal
+              host: localhost
+              port: 5432
+              user: skyportal
+              password: anything
+          EOF
+
+      - name: Initialize SkyPortal
+        run: |
+          # Usually, we create databases on the local machine, so
+          # `createdb` just works out of the box.  However, when doing
+          # this on a server, as in our case, we need to specify extra
+          # options.
+          #
+          # db_init should not complain if the databases exist already
+          #
+          echo "localhost:5432:*:skyportal:anything" > ~/.pgpass
+          chmod 600 ~/.pgpass
+
+          createdb -h localhost -U skyportal skyportal_test
+          psql -U skyportal -h localhost -c "GRANT ALL PRIVILEGES ON DATABASE skyportal_test TO skyportal;" skyportal_test
+
+          make db_init
+
+          pip list --format=columns
+          npm ls --depth 0
+
+          # Stamp Alembic HEAD
+          PYTHONPATH=. alembic -x config=config.yaml stamp head
+
+      - name: Test loading demo data
+        run: |
+          make run &
+          sleep 10 && make load_demo_data
+          kill %1
+
+      - name: Checkout PR branch
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          submodules: true
+          clean: false
+
+      - name: Check for migration branching
+        run: |
+          # Make sure latest revision on master is included in Alembic history
+          n_current_head=$( PYTHONPATH=. alembic history | grep -c $CURRENT_HEAD )
+          if [[ $n_current_head -eq 0 ]]
+          then
+            cat << EOF
+              The current head Alembic revision on master is not present in this branch's
+              Alembic history. Please merge in the latest commit on master.
+          EOF
+            exit 1
+          else
+            echo "Alembic history includes the latest on master."
+          fi
+
+          # Make sure there are not multiple heads to the Alembic history
+          num_heads=$( PYTHONPATH=. alembic history | grep -c '(head)' )
+          if [[ $num_heads -ne 1 ]]
+          then
+            cat << EOF
+              There is more than one head revision in this branch's Alembic history.
+              Please resolve the branching revisions.
+          EOF
+            exit 1
+          else
+            echo "Alembic history has no conflicts in branch."
+          fi
+
+      - name: Run migration
+        run: |
+          # Update dependencies that may break migrations since the first commit, i.e. deepdiff
```

Comment:
What does this comment mean?

Reply:
I ran into an error when attempting to run the migration here without updating the dependencies. It was an error associated with the deepdiff package and seemed to have to do with some breaking change between whatever version of deepdiff is installed per old requirements.txt compared to more recent commits.  I didn't dig into it the exact causes though.

Reply:
I see, sort of, I think :)

Maybe just comment this as `# Update to latest dependencies`?

Reply:
Sure, that makes sense